src/mcp-httpstream-server.ts

```
@@ -192,25 +192,133 @@ function registerTools() {
+    transport = transports[sessionId];
+    requestLogger.debug({ sessionId }, 'Reusing existing transport');
+  } else if (!sessionId) {
+    // Create new transport for initialization request
+    transport = new StreamableHTTPServerTransport({
+      sessionIdGenerator: () => randomUUID(),
+      enableJsonResponse: true, // Support both JSON and SSE
+      onsessioninitialized: (sessionId) => {
+        // Store the transport by session ID
+        transports[sessionId] = transport;
+        requestLogger.debug({ sessionId }, 'New session initialized');
+      }
```

ESLint

[error] Insert `,`

(prettier/prettier)